recipes/btllib/meta.yaml

```diff
@@ -16,26 +16,26 @@ requirements:
   build:
     - {{ compiler('cxx') }}
     - gcc >=6.0.0
-    - cmake
     - meson
     - ninja
+    - cmake
```

Comment:
Just curious, how did you decide on the order of packages?

Reply:
Oh, I just followed btllib's README, thought it'd be more consistent if we make changes in the future.